contracts/test/EIP1820.js

```diff
@@ -0,0 +1,40 @@
+
+describe('EIP 1820', () => {
+  if (isGwMainnetV1()) {
+    return;
+  }
+
+  let provider = waffle.provider;
+
+  describe('Deploy', () => {
+    it('transferred value and fee', async () => {
+      let code = await provider.getCode(contract);
+      expect(code).to.be.equal("0x", "account isn't deployed");
+      // Create a transaction object
+      let tx = {
+        to: deployer,
+        // Convert currency unit from ether to wei
+        value: ethers.utils.parseEther("0.08")
+      };
+      // Send a transaction
+      let txObj = await wallet.sendTransaction(tx);
+      console.log('prepare deployment ETH', txObj.hash)
+      // send the deployment transaction
+      const { hash } = await provider.sendTransaction(
```

Comment:
Does here will use non-eip155 tx ?

Reply:
https://eips.ethereum.org/EIPS/eip-1820#deployment-method

Yes, the binary transaction is defined in EIP-1820